### PR TITLE
Update static_routes delete operation

### PR DESCRIPTION
**Pull request by NilashishC**
Depends-On: https://github.com/ansible-collections/cisco.iosxr/pull/29

##### SUMMARY

- Update static routes delete operation
- Improve gitignore

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME
iosxr_static_routes

### CODE REVIEW

**Comment by NilashishC** [comment]
recheck

**Comment by NilashishC** [comment]
recheck

**Comment by NilashishC** [comment]
recheck

File: .gitignore [hunk 1]
```diff
@@ -1 +1,107 @@
+ld products...
```
Comment (rohitthakur2590) [inline .gitignore:1]: do we need the '.gitignore' file at top level here?  some stuff might be irrelevant for others. 
So locally we could use global gitignore as well .

Comment (NilashishC) [reply .gitignore:1]: This is copied over from ansible/ansible. Having a comprehensive gitignore saves everyone the trouble of updating it locally.

**Pull request merged**